airbyte_cdk/manifest_migrations/migration_handler.py

```diff
@@ -112,18 +116,30 @@ def _version_is_valid_for_migration(
         self,
         manifest_version: str,
         migration_version: str,
-    ) -> bool:
+    ) -> Tuple[bool, bool]:
         """
-        Checks if the given manifest version is less than or equal to the specified migration version.
+        Decide whether *manifest_version* satisfies the *migration_version* rule.
+
+        Rules
+        -----
+        1. ``"*"``
+           – Wildcard: anything matches.
+        2. String starts with a PEP 440 operator (``==``, ``!=``, ``<=``, ``>=``,
+           ``<``, ``>``, ``~=``, etc.)
+           – Treat *migration_version* as a SpecifierSet and test the manifest
+             version against it.
+        3. Plain version
+           – Interpret both strings as concrete versions and return
+             ``manifest_version <= migration_version``.
+        """
+        if migration_version == "*":
```

Comment:
I'd change this to a const and make it *.*.* so that the door is open to regex matches later.